core/services/form/meta/Element.php

```diff
@@ -40,6 +45,7 @@ public function __construct()
                 Element::STATUS_ACTIVE   => esc_html__('Active', 'event_espresso'),
                 Element::STATUS_ARCHIVED => esc_html__('Archived', 'event_espresso'),
                 Element::STATUS_DEFAULT  => esc_html__('Default', 'event_espresso'),
+                Element::STATUS_DEFAULT  => esc_html__('Shared', 'event_espresso'),
```

Comment:
keys were duplicated here by accident, but it's fixed

Reply:
Oops! Thank you